<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtTerminal</class>
 <widget class="QMainWindow" name="QtTerminal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QCommandEdit Demo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0" colspan="2">
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">background: rgb(255, 238, 153);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This is a demo of the &lt;span style=&quot; text-decoration: underline;&quot;&gt;QCommandEdit&lt;/span&gt; widget.&lt;/p&gt;&lt;p&gt;Use &lt;span style=&quot; font-weight:600;&quot;&gt;Up&lt;/span&gt;/&lt;span style=&quot; font-weight:600;&quot;&gt;Down&lt;/span&gt; to navigate history, or to search history; Use &lt;span style=&quot; font-weight:600;&quot;&gt;Tab&lt;/span&gt;/&lt;span style=&quot; font-weight:600;&quot;&gt;Shift+Tab&lt;/span&gt; to complete words.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <property name="margin">
       <number>16</number>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QCommandEdit" name="commandEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QTextEdit" name="textCmdLog">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QListWidget" name="listWords">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>355</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="labelListWords">
      <property name="text">
       <string>自定义指令</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="labelCmdLog">
      <property name="text">
       <string>历史记录：</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QTextBrowser" name="txb_output"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>769</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCommandEdit</class>
   <extends>QLineEdit</extends>
   <header>qcommandedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
